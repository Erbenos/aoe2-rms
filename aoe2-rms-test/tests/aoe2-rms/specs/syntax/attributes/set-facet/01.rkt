#lang aoe2-rms
; Generate 10 jungle straggler trees that all look identical and have the appearance of frame n_tree_jungle_x1_031 (a large and small palm tree)

<OBJECTS-GENERATION>
(create-object 'JUNGLETREE
               (number-of-objects 10)
               (set-facet 30))
