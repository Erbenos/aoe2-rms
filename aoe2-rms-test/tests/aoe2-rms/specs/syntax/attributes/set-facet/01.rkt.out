<OBJECTS_GENERATION>
create_object JUNGLETREE {
  number_of_objects 10
  set_facet 30
}
